scripts/serve-local.sh

```diff
@@ -9,6 +9,10 @@ docker run \
     --rm \
     --name janis \
     --tty --interactive \
-    --publish 3000:3000 \
-    --volume "$PWD:/app" \
+    --publish 3000:80 \
+    --volume "$PWD/src:/app/src" \
+    --volume "$PWD/public:/app/public" \
+    --volume "$PWD/package.json:/app/package.json" \
+    --env PORT=80 \
```

Comment:
@benweatherman please don't hate me for the back and forth in this PR. I blame the current administration. 
This seems like the cleanest route to set environment vars as .env files aren't acknowledged universally (ran into NODE_PATH issues with storybook on ctxfloods) and it doesn't litter up our package.json. Let me know if you think otherwise. I'm definitely out of my party zone.

Reply:
This seems like a great way to go too. It'll either work or we'll change it 😸